<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(43,49,55);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,0">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="deleteDataButton">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border-width:2px;
padding:6px;
border-color:rgb(255,0,0);
background-color:rgb(255,0,0);
color:rgb(255,255,255);
font-size:12pt;</string>
        </property>
        <property name="text">
         <string>Удалить все записи</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="generateNewPasswordButton">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border-width:2px;
padding:6px;
border-color:rgb(13,116,231);
background-color:rgb(13,116,231);
color:rgb(255,255,255);
font-size:12pt;</string>
        </property>
        <property name="text">
         <string>Сгенерировать новый пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addNote">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border-width:2px;
padding:6px;
border-color:rgb(47,187,79);
background-color:rgb(47,187,79);
color:rgb(255,255,255);
font-size:12pt;</string>
        </property>
        <property name="text">
         <string>Добавить запись</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quit">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border-width:2px;
padding:6px;
border-color:rgb(255,0,0);
background-color:rgb(255,0,0);
color:rgb(255,255,255);
font-size:12pt;</string>
        </property>
        <property name="text">
         <string>Выйти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="treeWidget">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255,255,255);border-width:2;
border-radius:6px;
background-color:rgb(36,41,45);
font-size:12
</string>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255,255,255);border-width:2;border-radius:5px;background-color:rgb(36,41,45);</string>
        </property>
        <item>
         <property name="text">
          <string>Записи  </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Пароли</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="settingsAction">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
